topKFrequentElements.js

```diff
@@ -0,0 +1,36 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/top-k-frequent-elements/
+
+const topKFrequent = (nums, k) => {
+  const numsOccurences = {};
+  for (let i = 0; i < nums.length; i += 1) {
+    if (!numsOccurences[nums[i]]) {
+      numsOccurences[nums[i]] = 1;
+    } else {
+      numsOccurences[nums[i]] += 1;
+    }
+  }
```

Comment:
Use range loop here - it'll get more readable.

Reply:
Refactored with [03d611b](https://github.com/Femalopper/tasks/pull/7/commits/03d611b9dec57d7079e9fd7faf9220e8d90cece0)